build/deps/postgres-documentdb.Dockerfile

```diff
@@ -2,4 +2,7 @@
 # FROM ghcr.io/ferretdb/postgres-documentdb:17-0.102.0-ferretdb-2.0.0
 
 # Use moving development image during development.
-FROM ghcr.io/ferretdb/postgres-documentdb-dev:17-ferretdb
+# FROM ghcr.io/ferretdb/postgres-documentdb-dev:17-ferretdb
+
+# FIXME
+FROM ghcr.io/ferretdb/postgres-documentdb-dev:17-pr-merge-into-ferretdb
```

Comment:
Once FerretDB/documentdb#40 is merged, we can use `17-ferretdb`.

Reply:
It is merged